installing/installing-preparing.adoc

```diff
@@ -125,14 +125,15 @@ endif::openshift-origin[]
 |
 |
 
-|Network Operator
+|Network Customization
 |xref:../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[X]
 |xref:../installing/installing_azure/installing-azure-network-customizations.adoc#installing-azure-network-customizations[X]
 |xref:../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[X]
 |
 |
 |
-|
+|xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations:[X]
```

Comment:
@GroceryBoyJr Can you remove colon at the end of the link? The PR is failing Travis checks.